docs/modules/ROOT/pages/vshn-managed/how-tos/logging.adoc

```diff
@@ -0,0 +1,57 @@
+= How-To Access Service Logs
+
+== Via CLI [[viacli]]
+
+Each service provided by VSHN is deployed in its own namespace.
+The access to those namespaces is deliberately limited, for more information see xref:references/permissions.adoc[].
+
+It is possible to list and view the logs of all pods in those namespaces.
+
+.Get the instance namespaces
+[source,bash]
+----
+oc get namespaces
+----
+
```

Comment:
Nitpick: We actually have the namespace in the `status` (at least for Postgres).  So maybe we could also add something like:

Reply:
yeah, it's currently only in the postgresql. But I guess this would be a good addition for redis as well.